### PR TITLE
NRB, mentor replacement, rubric, misc. fixes

### PR DESCRIPTION
**Pull request by kburk1997**
**Resolves Issue:** (add GitHub issue number here)
#55, #53, #40, #31
**Changes in this pull request**:
- Fix broken CoC links
- Add NRB documentation
- Merge in rubric with some slight changes:
  - Fix spelling errors
  - Fix run on sentences in the Code of Conduct Addendum
  - Rename "Addendum - Code of Conduct" to "Citizenship"
- Add draft mentor replacement policy

### CODE REVIEW

**Comment by kburk1997** [comment]
Also --- I'm not sure what to do for the sidebar search, if someone could help me out that'd be great

File: docs/index.html [hunk 1]
```diff
@@ -19,12 +19,17 @@
       loadNavbar: false,
       maxDepth: 3,
       subMaxLevel: 3,
-      basePath: 'https://raw.githubusercontent.com/rcos/rcos-handbook/master/docs',
+      //basePath: 'https://raw.githubusercontent.com/rcos/rcos-handbook/master/docs',
```
Comment (aeksco) [inline docs/index.html]: Oops! This needs to be uncommented for the site to work in production - but in development you need it commented out - it's pretty annoying :/

**Comment by aeksco** [comment]
@kburk1997 Sidebar search is a bit odd - haven't got much of a grasp on it myself, but the [docs are here](https://github.com/docsifyjs/docsify/blob/master/docs/plugins.md#full-text-search) if you want to try to tune it a bit more.

File: docs/index.html (hunk 1 above)
Comment (kburk1997) [reply docs/index.html]: Good catch!

**Pull request merged**